improver_tests/acceptance/test_weighted_blending.py

```diff
@@ -134,17 +134,36 @@ def test_percentile(tmp_path):
     acc.compare(output_path, kgo_path)
 
 
-@pytest.mark.slow
 def test_cycletime(tmp_path):
     """Test cycletime blending"""
     kgo_dir = acc.kgo_root() / "weighted_blending/cycletime"
     kgo_path = kgo_dir / "kgo.nc"
-    input_path = kgo_dir / "input.nc"
+    input_paths = sorted((kgo_dir.glob("input_temperature*.nc")))
+    print('inputs', input_paths)
```

Comment:
Need to remove this print statement. Also I think you need to exclude one of the input cubes or change the cycletime as one of the input cubes has the same cycletime as the cycletime you pass in.